### ISSUE TITLE
Mouse/touch input on macOS is offset in retina mode (RetroArch mode)

### ISSUE THREAD

**Issue by lxgr** [libretro/Mu#12]
When launching Retroarch in "retina" mode (the default), all clicks are offset by a factor of 2 horizontally and vertically (i.e. a tap in the center of the screen is scaled to the bottom right, and taps outside the upper left quadrant are ignored).

Everything works correctly when launching RetroArch in "low resolution" mode (by clicking "get info" in Finder on the .app).

Mu version 1.9.9 (v1.3.0 7ead066)

**Comment by XerTheSquirrel** [comment]
Does this happen with other touchscreen cores in RetroArch that you know of?

**Comment by lxgr** [comment]
@XerTheSquirrel  What would be a good one to test this with? NDS?

**Comment by XerTheSquirrel** [comment]
> 
> 
> @XerTheSquirrel What would be a good one to test this with? NDS?

Yes, would give NDS a try. If that works okay, then the issue would be with Mu.

**Comment by lxgr** [comment]
It happens in DeSmuME in both "low resolution" and standard mode!

**Comment by XerTheSquirrel** [comment]
> 
> 
> It (or something very similar) happens in DeSmuME in both "low resolution" and standard mode! Same for ScummVM. In both, I can see two cursors, one of them moving at a different speed than the system one.
> 
> However, in this case at least I can use one of these cursors (and just ignore the system one). For Mu, I can only see the system cursor which clicks at an offset.

Seems this would be a RetroArch issue, since the cursors should point to the same location so something weird is going on.
